### ISSUE TITLE
`setup-env` action is modifying my username

### ISSUE THREAD

**Issue by jason-edstrom** [browserstack/github-actions#24]
I noticed that when I use the `setup-env` action, my username gets appended with `-GitHubAction.` Do you know if this is intended behavior?

<img width="722" alt="image" src="https://github.com/browserstack/github-actions/assets/2515629/69d11232-8ad8-4a96-9587-a7eb41ee7cbf">
https://github.com/browserstack/github-actions/blob/63b648e87c6c2b48039b6e7af80b259c60d85f99/setup-env/src/actionInput/inputValidator.js#L79

**Comment by francisf** [comment]
Yes. It is used for instrumentation around usage of GH Actions (or any other CI system). This shouldn't impact your tests in any way though.
